### ISSUE TITLE
ACC+DAT+GEN should be split into 3 separate entries

### ISSUE THREAD

**Issue by ivri** [unimorph/nds#3]

**Comment by kbatsuren** [comment]
@ivri, sure I will make a time in June for fixing all the issues recently raised.

**Comment by kbatsuren** [comment]
@Munkh99, could you help us to split 'ACC+DAT+GEN' into three different entries and submit PR? 

For instance,

```
gammlig	en gammligsten	ADJ;ACC+DAT+GEN;SPRL;INDF;MASC;SG
```

should be split as follows: 

```
gammlig	en gammligsten	ADJ;ACC;SPRL;INDF;MASC;SG
gammlig	en gammligsten	ADJ;DAT;SPRL;INDF;MASC;SG
gammlig	en gammligsten	ADJ;GEN;SPRL;INDF;MASC;SG
```

Please let me know if you have questions.

**Comment by kbatsuren** [comment]
This issue is fixed in pull request #4. Thanks for @Munkh99.
